bee-network/bee-autopeering/src/time.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::{SystemTime, UNIX_EPOCH};
+
+pub(crate) type Timestamp = u64;
+pub(crate) type Timespan = u64;
+
+pub(crate) const SECOND: u64 = 1;
```

Comment:
I'll add a comment explaining that all of these are measured in seconds.

Reply:
I don't understand why that is not obvious? You want me to add that SECOND = 1, mean, that `1` represents a second?

Reply:
I see no reason to believe that all the constants are measured in seconds, milliseconds, nanoseconds or any other unit. In particular, when I saw an expression like `Duration::from_secs(5 * SECONDS)` I wasn't sure what was the purpose of `SECONDS` there so I started wondering if there was something else about it that I didn't understood, so being able to hover over any of the constants and seeing a comment like: `A <whatever> measured in seconds` would clear things up.

Reply:
Done